library/Fission/Error/AlreadyExists/Types.hs

```diff
@@ -21,6 +21,9 @@ instance Display (AlreadyExists entity) => ToServerError (AlreadyExists entity)
 instance Display (AlreadyExists User) where
   display _ = "User already exists"
 
+instance Display (AlreadyExists UserChallenge) where
+  display _ = "Challenge already exists for email"
```

Comment:
Is it scoped per email address? My understanding was that they're assumed globally unique so we can look them up by the challenge text only, no?

Reply:
oh yup I'll change the wording :+1: I was originally handling this differently and never changed the wording

yup they're assumed globally unique. we also shouldn't ever hit this error, it'd be about the same probability as a SHA1 collision